packages/core/addon/components/navi-visualization-config/table.hbs

```diff
@@ -11,23 +11,21 @@
       {{!-- Toggle for grand total --}}
       <div class="table-config__totals-toggle-section">
         <span class="table-config__totals-toggle-label">Grand Total</span>
-        <XToggle
+        <DenaliSwitch
           class="table-config__total-toggle-button table-config__total-toggle-button--grand-total"
-          @value={{readonly @options.showTotals.grandTotal}}
-          @size="small"
-          @onToggle={{this.onToggleGrandTotal}}
+          @isEnabled={{@options.showTotals.grandTotal}}
+          @onChange={{pipe (pick 'target.value') (fn this.onToggleGrandTotal )}}
         />
       </div>
 
       {{!-- Toggle for subtotal --}}
       {{#if @request.dimensionColumns.length}}
         <div class="table-config__totals-toggle-section">
           <span class="table-config__totals-toggle-label">Subtotal</span>
-          <XToggle
+          <DenaliSwitch
             class="table-config__total-toggle-button table-config__total-toggle-button--subtotal"
-            @value={{readonly this.showSubtotalDropdown}}
-            @size="small"
-            @onToggle={{this.onToggleSubtotal}}
+            @isEnabled={{this.showSubtotalDropdown}}
+            @onChange={{pipe (pick 'target.value') (fn this.onToggleSubtotal )}}
```

Comment:
```suggestion
            @onChange={{pipe (pick "target.value") (fn this.onToggleSubtotal )}}
```

Reply:
do you want me to update all the 'target.values's? All the others are single quotes

Reply:
Yea we try to use double quotes in hbs files so go ahead and update those